guidelines/monai-application-package.md

```diff
@@ -1,498 +1,588 @@
 
-A MAP MUST be in a format which supports local execution in a development environment.
+# Architecture & Design
 
-> [Note]
-> See [MONAI Operating Environments](monai-operating-environments.md) for additional information about supported environments.
+## Description
 
+The MONAI Application Package (MAP) is a functional package designed to act on datasets of a prescribed format. A MAP is a container image that adheres to the specification provided in this document.
 
-### Containerization
+## Application
 
-A MAP SHALL be a containerized application to maximize portability of its application.
+The primary component of a MAP is the application. The application is provided by an application developer and incorporated into the MAP using the MONAI Deploy Application Packager.
 
+All application code and binaries SHALL be in the `/opt/holoscan/app/` folder, except for any dependencies installed by the MONAI Deploy Packager during the creation of the MAP.
 
-### Compatible with Kubernetes
+All AI models (PyTorch, TensorFlow, TensorRT, etc.) SHOULD be in separate sub-folders of the `/opt/holoscan/models/` folder. In specific use cases where the app package developer is prevented from enclosing the model files in the package/container due to intellectual property concerns, the models can be supplied from the host system when the app package is run, e.g., via the volume mount mappings and the use of container env variables.
 
-A MAP SHALL NOT be in a format which inhibits or hampers the ability to deploy it using Kubernetes.
+## Manifests
 
+A MAP SHALL contain two manifests: the application manifest and the package manifest. The package manifest shall be stored in `/etc/holoscan/pkg.json`, and the application manifest shall be stored in `/etc/holoscan/app.json`. Once a MAP is created, its manifests are immutable.
 
-### OCI Compliance
+### Application Manifest
 
-The containerized portion of a MAP SHALL comply with [Open Container Initiative](https://opencontainers.org/) format standards.
+The Application Manifest file provides information about the MAP's Application.
 
+- Application Manifest MUST define the type of the containerized application (`/etc/holoscan/app.json#type`).
 
-### Facilitate GPU Acceleration
+  - Type SHALL have the value of either `service` or `application.`
 
-A MAP SHALL enable applications to be developed with GPU acceleration.
+- Application Manifest MUST define the command used to run the Application (`/etc/holoscan/app.json#command`).
 
-A MAP SHALL NOT impede the adoption or utilization of accelerator (GPU) technology.
+  - When not provided, the MAP will be considered invalid and not runnable by MONAI Deploy Application Runner and hosting services that comply to MAP specification.
 
+- Application Manifest SHOULD define the version of the manifest file schema (`/etc/holoscan/app.json#api-version`).
```

Comment:
I would include a 'The' or 'An' or make Application Manifest plural. e.g.
`An Application Manifest SHOULD...`
or
`Application Manifests SHOULD...`

Same goes for similar examples in this doc.